website/docs/r/virtual_network.html.markdown

```diff
@@ -99,6 +101,14 @@ A `ddos_protection_plan` block supports the following:
 
 ---
 
+A `encryption` block supports the following:
+
+* `enabled` - (Required) Enable/disable encryption on Virtual Network.
+
+* `unencrypted_allowed` - (Optional) Whether ths virtual network allos VM that does not support encryption. value `false` for `DropUnencrypted`. value `true` for `AllowUnencrypted`. Defaults to `false`.
```

Comment:
We should expose this as the string constant, not as a boolean, since it's likely other values will get added here in the future

Reply:
thanks! document updated.